tests/tests_pytest.yaml

```diff
@@ -0,0 +1,183 @@
+TestAdaptahop:
```

Comment:
I don't think this file is necessary at all. For nose that was a way to pseudo parallelize tests, but pytest-xdist handles that without such tricks. Both yaml and h5 file names could be autogenerated based on test class name.

Reply:
That would be quite nice.

Reply:
I removed the file and now have the test names generated from the test class. The only thing I didn't do was get the test version number (e.g., frontend_000.yaml), so right now the answer files are TestClassName.yaml and TestClassName.h5.

Reply:
We still need some sort of tag for new answer generation, but you could use e.g. attribute on a class or something like that.

Reply:
Sounds good. Should that go in this PR or a future one?

Reply:
Probably this one since it's an infrastructure PR and later ones I think are the actual tests.

Reply:
Ok. Added.